oktapam/convert/security_policy_privileges.go

```diff
@@ -78,7 +76,8 @@ func SecurityPolicyRulePrivilegeContainerFromSDKToModel(ctx context.Context, in
 		out.PasswordCheckoutSSHPrivilege = privilege
 
 	} else {
-		panic("missing stanza in SecurityPolicyRulePrivilegeContainerFromSDKToModel")
+		diags.AddError("missing stanza in OktaPAM provider", "missing stanza in SecurityPolicyRulePrivilegeContainerFromSDKToModel")
```

Comment:
I don't quite understand this error. This is when SDK -> Model so API response is missing the required field but error says missing stanza in oktapam provider.

Reply:
If we add a new privilege value, and a human changes the privileges in the UI for an existing policy to one not supported by the Terraform provider, the provider won't know how to convert it to the model.